docs/faq.md

```diff
@@ -11,32 +11,30 @@ For value transactions, there is the wallet.rs library which provides a stateful
 
-## What is Dust protection and how does it work?
+## What is Dust Protection and How Does it Work?
 
-Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this we only allow microtransaction below 1Mi of IOTA tokens to another address if you already have at least 1Mi on that address.
+Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this, we only allow microtransaction below 1Mi of IOTA tokens to another address if you already have at least 1Mi on that address.
 
 You can read more about Dust Protection [in the RFC here](https://github.com/iotaledger/protocol-rfcs/pull/32).
 
-## What's happening with Coordicide?
+## What is Happening with Coordicide?
 
 This release called Chrysalis still depends on the Coordinator/Compass to run and issue milestones. The next big release after Chrysalis will be Coordicide where we get rid of that dependency. This release is currently in a research/testing phase and will be the main priority after the Chrysalis release.
 
-## When is a transaction on the network considered final/irreversible?
+## When is a Transaction on the Network Considered Final/Irreversible?
 
 Confirmation times on the new network are on average around 10 seconds. Once a transaction is set to confirm it’s final transaction, you don’t have block confirmations like with blockchain.
 
-## Hornet or Bee? Which node software should I use?
+## Hornet or Bee? Which Node Software Should I Use?
 
-You can pick either Bee (Rust based) or Hornet (Go based). We currently recommend Hornet since it’s the more complete version of the node software, Bee is still missing some optional features you might wish to use.
+You can either pick Bee (Rust based) or Hornet (Go based). We currently recommend Hornet since it’s the more complete version of the node software, Bee is still missing some optional features you might wish to use.
 
-## What are spent addresses and why are they dangerous?
+## What are Spent Addresses and Why are they Dangerous?
 - In the IOTA 1.0 network, IOTA used Winternitz One Time Signatures (W-OTS) - think of these as, more or less, an authenticator and validator for a transaction. These keys and signatures are highly secure against malicious attacks for signing transactions. But, on the downside, by signing a transaction, W-OTS reveals parts of a private key for the specific address tokens are being spent from. 
 - With W-OTS every time a signature is signed to spend tokens from a particular address, any remaining tokens need to be moved onto a new address to prevent malicious actors from brute-forcing (trial-and-error guessing) the remaining parts of the private key for the address. That’s the main reason why this signature scheme is considered to be a “one-time signature”. 
```

Comment:
- "a signature is signed" seems a bit redundant.

Reply:
Updated!